acoustid/api/v2/misc.py

```
@@ -4,7 +4,7 @@
 import logging
 
 import requests
-from sqlalchemy import sql
+from sqlalchemy import BinaryExpression, sql
```

Ruff (0.8.2)

`sqlalchemy.BinaryExpression` imported but unused

Remove unused import: `sqlalchemy.BinaryExpression`

(F401)